modules/tinymce/src/plugins/table/main/ts/ui/CellDialog.ts

```diff
@@ -45,40 +70,46 @@ const updateAdvancedProps = (modifier: DomModifier, data: CellData) => {
 // how as part of this, it doesn't remove any original alignment before
 // applying any specified alignment.
 
-const applyCellData = (editor: Editor, cells: HTMLTableCellElement[], data: CellData) => {
+const applyCellData = (editor: Editor, cells: SugarElement<HTMLTableCellElement>[], data: CellData) => {
   const dom = editor.dom;
   const isSingleCell = cells.length === 1;
 
-  Arr.each(cells, (cell) => {
-    // Switch cell type if applicable
-    const cellElm = data.celltype && Util.getNodeName(cell) !== data.celltype ? (dom.rename(cell, data.celltype) as HTMLTableCellElement) : cell;
-    const modifier = isSingleCell ? DomModifier.normal(editor, cellElm) : DomModifier.ifTruthy(editor, cellElm);
-
-    updateSimpleProps(modifier, data);
-
-    if (hasAdvancedCellTab(editor)) {
-      updateAdvancedProps(modifier, data);
-    }
-
-    // Remove alignment
-    if (isSingleCell) {
-      Styles.unApplyAlign(editor, cellElm);
-      Styles.unApplyVAlign(editor, cellElm);
-    }
-
-    // Apply alignment
-    if (data.halign) {
-      Styles.applyAlign(editor, cellElm, data.halign);
-    }
-
-    // Apply vertical alignment
-    if (data.valign) {
-      Styles.applyVAlign(editor, cellElm, data.valign);
-    }
-  });
+  if (cells.length >= 1) {
```

Comment:
Since we're doing this now, anything below that is wrapped in a `isSingleCell` `if` won't be called? So, why are we doing this now?

Reply:
Also what happens now if it's just one cell?

Reply:
Nvm, I misread!